core/lib/zksync_core/src/tee_verifier_input_producer/mod.rs

```diff
@@ -0,0 +1,262 @@
+//! Produces input for a TEE Verifier
```

Comment:
Request: currently we're working on extracting all the modules from `core/lib/zksync_core` to standalone crates in `core/node`. Could you, please, do the same with this module, so that the scope of remaining work doesn't increase?

If not, no problem, I'll do it myself once this is merged.